src/Checkbox/README.md

```diff
@@ -0,0 +1,24 @@
+# Checkbox
+To implement Checkbox component into your project you'll need to add the import:
+```jsx
+import Checkbox from "@kiwicom/orbit-components/lib/Checkbox";
+```
+After adding import into your project you can use it simply like:
+```jsx
+<Checkbox />
+```
+## :rocket: Props
+Table below contains all type of the props available in Checkbox component.
+
+| Name         | Type         | Default | Description                      |
+| :-------     | :----------- | :------ | :------------------------------- |
+| checked      | `boolean`    | `false` | If `true`, the Checkbox will be checked.
+| disabled     | `boolean`    | `false` | If `true`, the Checkbox will be set up as disabled.
+| hasError     | `boolean`    | `false` | If `true`, error border will be set up around the Checkbox. [See Functional specs](#warning-functional-specs) :eyes:
```

Comment:
If `true`, the border of the Checkbox will turn red.

Same for the `<Radio />`.